csbmysql/testfixtures/ssl_mysql/keys/client.key

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
question: Why do we need the extra line at the top of the file?

Reply:
Good question. Golang was throwing this error: `tls: failed to find any PEM data in key input`.

There is an issue with the key file like UTF-8 BOM at the start of the file or similar. I've seen people get errors because additional bytes were added at the end of the file, etc, but I am not sure exactly what it is the reason of this error.